lighthouse-cli/bin.ts

```diff
@@ -143,6 +147,10 @@ if (!cliFlags.outputPath && cliFlags['output-path']) {
   cliFlags.outputPath = cliFlags['output-path'];
 }
 
+if (!cliFlags.maxWaitForLoad && cliFlags['max-wait-for-load']) {
```

Comment:
Let's bump yargs from our 3.30.0 to 3.32.0.
It has the fix for these: https://github.com/yargs/yargs/commit/51c0063183486193c92117d2bdcc61a67387f3d4